helix-term/src/commands.rs

```diff
@@ -1138,15 +1139,19 @@ mod cmd {
             doc.indent_style = s;
         } else {
             // Invalid argument.
-            editor.set_error(format!("invalid indent style '{}'", args[0],));
+            write!(
+                &mut editor.error_mut(),
```

Comment:
By the way you don't need to borrow again here. `error_mut()` already returns a `&mut String`.

Reply:
I think I tried that but it didn't work. But weird, now seemed to work.